apps/trip_plan/lib/trip_plan/api/open_trip_planner/parser.ex

```diff
@@ -68,15 +68,26 @@ defmodule TripPlan.Api.OpenTripPlanner.Parser do
   defp error_message_atom("NO_STOPS_IN_RANGE", _opts), do: :location_not_accessible
   defp error_message_atom(_, _opts), do: :unknown
 
-  defp parse_itinerary(json, request_params) do
+  defp parse_itinerary(json) do
     %Itinerary{
       start: parse_time(json["startTime"]),
       stop: parse_time(json["endTime"]),
       legs: Enum.map(json["legs"], &parse_leg/1),
-      accessible?: request_params["wheelchair"] == "true"
+      accessible?: parse_float(json["accessibilityScore"]) > 0.5
```

Comment:
👁️ I just arbitrarily picked `0.5`. I'm not sure what constitutes an accessible itinerary.

Reply:
So I checked and accessible route and it returned 1.  I am wondering if anything less than 1 means part of the trip might not be accessible.

Reply:
The docs say it is a score between 0 and 1. If you check the Orange Line it comes back with 1 because all stops are wheelchair accessible. I imagine we will have to check a bunch of different itineraries to figure out what a "good" score looks like. But, for now, this gets itineraries showing as accessible.

Reply:
What I am trying to get at is if it returns .75 and one part of the trip isn't accessible are we mis leading the riders.  It might be safe to just keep the check at 1, until more research can be done